app-emulation/bochs/bochs-2.6.9.recipe

```diff
@@ -1,29 +1,30 @@
 SUMMARY="The open-source IA-32 emulator"
 DESCRIPTION="Bochs IA-32 Emulator provides a virtual PC that can run operating systems such as Windows, Linux, and BSD"
-HOMEPAGE="http://bochs.org/"
-COPYRIGHT="2001-2006 The Bochs Project"
-LICENSE="GNU LGPL v2.1"
-REVISION="3"
-SOURCE_URI="http://downloads.sourceforge.net/project/bochs/bochs/2.6/bochs-2.6.tar.gz"
-CHECKSUM_SHA256="65123e44a6d2c1148d741da025d16311b9ccd600ef268b911aba73c67eaf86e1"
+HOMEPAGE="http://bochs.sourceforge.net/"
+COPYRIGHT="2001-2017 The Bochs Project"
+LICENSE="GNU LGPL v2.1
+	BOCHS"
```

Comment:
```diff
 LICENSE="GNU LGPL v2.1
-	BOCHS"
+	Bochs"
```
I think "Bochs" is better than "BOCHS" because it's not an acronym.
(If you change this, don't forget to rename the file itself.)

Everything else is OK.
(I have built the recipe on all possible architectures and the app seems to be working fine. Well, I started bochs but did not provide any condifuration, but I'm sure the recipe is OK.)